### PR TITLE
Add vim syntax highlighting for *uts files

### PR DESCRIPTION
**Pull request by polybassa**
As discussed in #2075

### CODE REVIEW

**Comment by guedou** [comment]
Thanks for the PR. Could you remove the README and add the text to Scapy documentation? That will make this syntax file more visible.

**Comment by gpotter2** [comment]
IMH https://github.com/secdev/scapy/blob/master/doc/scapy/development.rst
Would be a good place.

**Comment by polybassa** [comment]
Thanks for your advice. Did it.

File: doc/scapy/development.rst [hunk 1]
```diff
@@ -253,3 +253,19 @@ For example, on a fresh Debian installation, the following command will start
...
+
+If ftdetect/filetype.vim already exists, you might need to modify this file manually.
+
+These commands will do the installation::
+
+ cp -i -v ftdetect/filetype.vim $HOME/.vim/ftdetect/filetype.vim
+ cp -i -v ftdetect/uts.vim $HOME/.vim/ftdetect/uts.vim
+ cp -i -v syntax/uts.vim $HOME/.vim/syntax/uts.vim
+
+Alternatively, a install script in ``scapy/doc/syntax/vim_uts_syntax/`` does the installation automatically.
```
Comment (guedou) [inline doc/scapy/development.rst]: -> Alternatively, the ``scapy/doc/syntax/vim_uts_syntax/install.sh`` shell script does the installation automatically.

File: doc/scapy/development.rst [hunk 2]
```diff
@@ -253,3 +253,19 @@ For example, on a fresh Debian installation, the following command will start
 all Scapy unit tests automatically without any external dependency::
 
  tox -- -K vcan_socket -K tcpdump -K tshark -K nmap -K manufdb -K crypto
+
+VIM syntax highlighting for *.uts files
```
Comment (guedou) [inline doc/scapy/development.rst]: Sphinx complains with ```scapy/doc/syntax/vim_uts_syntax/`` does the installation automatically.` You can fix this by removing the *.

Locally, `tox -l docs` will run the test on the documentation.

**Pull request merged**